content/resources/getting-started-with-infrastructure-as-code-aws/index.md

```diff
@@ -60,9 +60,9 @@ main:
 
     # The webinar presenters
     presenters:
-        - name: Diana Esteves
-          role: Solutions Architect, Pulumi
-          photo: /images/team/diana-esteves.jpg
+        - name: TBD
+          role: TBD
+          photo: TBD
```

Comment:
```suggestion
```

Reply:
Removing the photo entry ensures the anonymous platypus image is used as the avatar